modules/ROOT/partials/menu-item-ids/math-menu-items.adoc

```diff
@@ -0,0 +1,5 @@
+[cols="1,1,2",options="header"]
+|===
+|Menu item identifier |xref:menus-configuration-options.adoc#example-the-tinymce-default-menu-items[Default Menu Location] |Description
+|`+math+` |Insert |Provides a way for users to insert a `+<math>+` element into {productname}.
```

Comment:
The description should be consistent with the toolbar button description. I think a basic description is fine
```suggestion
|`+math+` |Insert |Opens the math modal dialog.
```